doc/1.4/language.md

```diff
@@ -931,6 +931,8 @@ machine objects, by letting users create FSMs by declaring group objects
 instantiating that template. FSM states can also be represented through a
 template instantiated by groups.
 
+#### Finite State Machines using groups
+
 The following demonstrates a simple example of how such an abstraction may be
```

Comment:
Need to also rephrase, "such an" now references to the other side of a heading. Perhaps part of the previous sentence should be moved to this side.